packages/common/src/index.ts

```diff
@@ -345,18 +345,43 @@ export default class Common {
   }
 
   /**
-   * True if block number provided is the hardfork (given or set) change block of the current chain
+   * True if block number provided is the hardfork (given or set) change block
    * @param blockNumber Number of the block to check
    * @param hardfork Hardfork name, optional if HF set
    * @returns True if blockNumber is HF block
    */
   isHardforkBlock(blockNumber: number, hardfork?: string): boolean {
     hardfork = this._chooseHardfork(hardfork, false)
-    if (this.hardforkBlock(hardfork) === blockNumber) {
-      return true
-    } else {
-      return false
-    }
+    return this.hardforkBlock(hardfork) === blockNumber ? true : false
```

Comment:
Ah the `? true : false` thing is unnecessary here too.

Reply:
Updated.